akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -312,6 +317,18 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     aeron = Aeron.connect(ctx)
   }
 
+  private def startAeronErrorLog(): Unit = {
+    val errorLog = new AeronErrorLog(new File(driver.aeronDirectoryName, CncFileDescriptor.CNC_FILE))
+    val lastTimestamp = new AtomicLong(0L)
+    import system.dispatcher // FIXME perhaps use another dispatcher for this
+    aeronErrorLogTask = system.scheduler.schedule(3.seconds, 5.seconds) {
+      if (!isShutdown) {
+        val newLastTimestamp = errorLog.logErrors(log, lastTimestamp.get)
+        lastTimestamp.set(newLastTimestamp + 1)
+      }
+    }
+  }
```

Comment:
So, Aeron writes errors to the shared memory file, and we poll it periodically to pull the entries into the Akka log?